dataland-backend/src/main/kotlin/org/dataland/datalandbackend/controller/CompanyDataController.kt

```diff
@@ -50,16 +57,39 @@ class CompanyDataController(
                 CompanySearchFilter(
                     searchString ?: "",
                     onlyCompanyNames,
-                    dataTypes ?: setOf(),
+                    dataTypes,
                     countryCodes ?: setOf(),
                     sectors ?: setOf(),
```

Comment:
Ich denke das `setOf()` ist nicht notwendig, weil die defaults im model von  `CompanySearchFilter` selbst bereits `setOf()` sind.

Reply:
Aber wir brauchen die setOfs doch hier, weil null ja nicht gesetzt werden darf.

Reply:
DIscussed with @fsoffel    StoredCompanySearchFilter and CompanySearchFilter shall be merged to one model which serves all needs

Reply:
Which is done.